### PR TITLE
Create how-to-contribute-to-open-source-projects-as-a-non-coder.md

### LINKED ISSUE #124: Cross blog this blog post
### Idea
In this article, I will share with you what I discovered and also mention some non-code ways you can contribute, open-source projects you can contribute to, and helpful resources.

### Topics

**Open Source**: 

**Beginner guide, non-tech, Open source**: 

### Cross-posting?

 https://benny.hashnode.dev/how-to-contribute-to-open-source-projects-as-a-non-coder

### Remarks

 My Social media: [Twitter](https://twitter.com/bennykillua) ,  [Linkedin](https://www.linkedin.com/in/ifeanyi-iheagwara/)

### PR DESCRIPTION
**Pull request by Bennykillua**
## Fixes Issue
 Closes #124 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: _drafts/how-to-contribute-to-open-source-projects-as-a-non-coder.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+- Community building.
+- **Compensation:** There are rarely any direct monetary incentives, however, programs such as [Google Season of Docs](https://developers.google.com/season-of-docs), [Google Summer of Code](https://summerofcode.withgoogle.com/), [Hyperledger Mentorship Program](https://wiki.hyperledger.org/display/INTERN), [Outreachy](https://www.outreachy.org/), and the [Open Collective bounty program](https://docs.opencollective.com/help/contributing/development/bounties) pay you to contribute.
+
+# Open Source Projects
+
+![where can i find oen source projects](https://cdn.hashnode.com/res/hashnode/image/upload/v1634909151136/IR4gF5koo.png)
+
+Today, there are a lot of open-source projects you can contribute to and it is up to you to choose the ones that interest you. Below are a few open-source projects you can choose from; take your time to go through each resource and see which one appeals to you.
+
+- [AWS](https://aws.amazon.com/blogs/aws/aws-documentation-is-now-open-source-and-on-github/?utm_source=feedburner&utm_medium=email&utm_campaign=Feed%3A+AmazonWebServicesBlog+%28Amazon+Web+Services+Blog%29)
```
Comment (Posandu) [inline _drafts/how-to-contribute-to-open-source-projects-as-a-non-coder.md:86]: umm there is some tracking code here. lol

Comment (Bennykillua) [reply _drafts/how-to-contribute-to-open-source-projects-as-a-non-coder.md]: code? just links

Comment (shockz09) [reply _drafts/how-to-contribute-to-open-source-projects-as-a-non-coder.md:86]: guess he took the link from somewhere and didn't remove it , @Bennykillua fix pls

Comment (Bennykillua) [reply _drafts/how-to-contribute-to-open-source-projects-as-a-non-coder.md]: Oh fixed. thanks @shockz09 @Posandu

**Comment by gouravkhunger** [comment]
Hey @Bennykillua, The PR checks are failing because your fork isn't up to date with the repository.

image

Could you fetch and merge with the main repository the changes and push them here?

[This might help](https://genicsblog.com/gouravkhunger/how-to-create-a-pull-request-in-github-correctly#optional-useful-information). That's also one of the reasons not to commit directly to the main branch of a fork 😄 

And regarding the small fixes in the links, it's alright. I'll fix them before publishing :)

**Comment by Bennykillua** [comment]
Thanks @gouravkhunger That article was really helpful. I have also fixed the issue with the link .

**Pull request merged**